### ISSUE TITLE
Add Open Graph meta tags

### ISSUE THREAD

**Issue by dokterbob** [ipfs-search/ipfs-search-frontend#21]
https://richpreview.com/?url=https://ipfs-search.com/

**Comment by madnificent** [comment]
Can you add suggestions for the static content to be placed in here?  We can probably annotate more content in our page semantically.  This set is easy to add.

Let's consider this story for static content and create a new one with dynamic content once this has been implemented.

**Comment by dokterbob** [comment]
@vandrongelen Feel free to suggest an alternative OG preview image.

**Comment by madnificent** [comment]
Yes, please suggest another picture or upload/pin the previous one.  The current ipfs link doesn't resolve here anymore :-)

**Comment by mehran-naghizadeh** [comment]
> https://richpreview.com

This website is not working.
